homeassistant/components/samsungtv/bridge.py

```diff
@@ -300,28 +300,30 @@ def try_connect(self) -> str:
             }
 
             result = None
-            try:
-                LOGGER.debug("Try config: %s", config)
-                with SamsungTVWS(
-                    host=self.host,
-                    port=self.port,
-                    token=self.token,
-                    timeout=config[CONF_TIMEOUT],
-                    name=config[CONF_NAME],
-                ) as remote:
+            LOGGER.debug("Try config: %s", config)
+            with SamsungTVWS(
```

Comment:
The context manager suggests there's a side effect here. Do we not need to catch any errors here?

Reply:
@MartinHjelmare I have reverted the changes in `try_connect`, to keep only the changes in `get_remote`